source/core/VirtualInstrument.cpp

```diff
@@ -34,7 +34,7 @@ NIError VirtualInstrument::Init(TypeManagerRef tm, Int32 clumpCount, TypeRef par
     _locals->SetElementType(localsType, false);
     _clumps->Resize1D(clumpCount);
     _lineNumberBase = lineNumberBase;
-    _clumpSource = clumpSource;
```

Comment:
I'm confused, this seems something that the compiler should catch. Are we ignoring warnings?